src/main/scala/io/iohk/ethereum/vm/OpCode.scala

```diff
@@ -834,9 +834,9 @@ sealed abstract class CallOp(code: Int, delta: Int, alpha: Int) extends OpCode(c
   }
 
   private def gasCap[W <: WorldStateProxy[W, S], S <: Storage[S]](state: ProgramState[W, S], g: UInt256, gExtra: UInt256): UInt256 = {
-   if (state.gas >= gExtra)
+    if (state.config.subGasCapDivisor.isDefined && state.gas >= gExtra)
```

Comment:
I don't like that we're checking `subGasCapDivisor` in two places (here and inside `state.config.gasCap`). Do you think we can make it just once? Not a blocker of course, but would be nice to have.